server/src/controllers/aplus.ts

```diff
@@ -122,7 +126,7 @@ export const addAplusGradeSources = async (
 /**
  * Responds with NewGrade[]
  *
- * @throws ApiError(400|404|409|422|502)
+ * @throws ApiError(400|404|409|502)
```

Comment:
Did you update the api documentation?

Reply:
This comment was outdated, 422 was never returned.